lib/scss/stanford_person.views.scss

```diff
@@ -100,6 +125,18 @@
   }
 }
 
+// Grid removal for people lists in cols
+.ptype-stanford-lists {
```

Comment:
this selector and its children arent part of this module. this should go in the stanford_profile_helper module.

Reply:
I moved it into this pr. https://github.com/SU-SWS/stanford_profile_helper/pull/118